### PR TITLE
Include process for status-go PRs

### PR DESCRIPTION
**Pull request by fryorcraken**
Define a process for status-go PRs as more of those are expected to be opened by the Waku team.

### CODE REVIEW

**Review by jakubgs — APPROVED** [review]

**Review by chaitanyaprem — APPROVED** [review]
LGTM

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -70,3 +70,18 @@ Documentation change PR (review please): https://github.com/status-im/status.im/
...
+- Specify potentially impacted user flows in _Areas that maybe impacted*.
+- Ensure that _Steps to test_ is filled in.
+
+### Risk
+
+Described potential risks and worst case scenarios.
+
+Tick **one**:
+- [ ] Low risk: 2 devs must perform testing as specified above and attach their results as comments to this PR.
+- [ ] High risk: QA team perform additional testing in the specified affected areas.
```
Comment (churik) [inline .github/PULL_REQUEST_TEMPLATE.md]: I'd add here a note that merging should not be allowed for High-risk PRs without QA approves it.

**Review by churik — APPROVED** [review]

**Review by plopezlpz — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (fryorcraken) [reply .github/PULL_REQUEST_TEMPLATE.md]: done!

**Comment by fryorcraken** [comment]
@cammellos ready for merging :)

**Pull request merged**